types/testingutils/keymanager.go

```diff
@@ -25,11 +27,34 @@ type testingKeyManager struct {
 	slashableDataRoots [][]byte
 }
 
+var (
+	instancesMap = make(map[string]*testingKeyManager)
+	mu           sync.Mutex
+)
```

Comment:
generally I would say global state is a bad idea, but since this is a test generation code and you are protecting it properly, its seems ok to me.